src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -452,7 +452,7 @@ subroutine get_fv3_regional_ensperts_run(this,en_perts,nelen,ps_bar)
 
               select case (trim(cvars3d(ic3)))
  
-                 case('sf','SF')
+                 case('sf','SF','u','U')
```

Comment:
Modification to use u/v in calculation of ensemble B is required not only in this file but also in ensctl2state.f90, ensctl2state_ad.f90, ensctl2model.f90, ensctl2model_ad.f90 and write_fv3_spread.f90.